R/build_collection.R

```diff
@@ -1,27 +1,165 @@
 
 
-build_collection <- function(title,
-                             links = build_links(),
-                             assets = NULL,
-                             extent) {
+build_collection <- function(id,
+                             links = NULL,
+                             title,
+                             assets = build_assets(tnail_object, tnail_title, parquet_uri, p_title, p_description),
```

Comment:
line 6 can't reference variables like `tnail_object` that aren't defined.  This should probably be 

```
assets = NULL
```

because technically `assets` is not marked as "REQUIRED" in https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md.  

alternately it could be something like

```
assets = build_assets()
```

i.e. calling the next constructor with no arguments.